### PR TITLE
Restructure roadmap so that we can link to sub-parts

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
So we want to point to specific subsections in the roadmap from the Working Group pages.
This requires the use of subtitles rather than a long list of items and sub-items.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

ROADMAP

##### ANSIBLE VERSION

v2.4

### CODE REVIEW

File: docs/docsite/rst/roadmap/ROADMAP_2_4.rst [hunk 1]
```diff
@@ -1,224 +1,220 @@
...
-
-    - Windows Server 2016 Integration Tests
-
-      - Restore Windows Server 2016 integration tests on Shippable.
-
-        - Originally enabled during the 2.3 release cycle, but later disabled due to intermittent WinRM issues.
-        - Depends on resolution of WinRM connection issues.
+============================
+**Target: Aug/Mid-September 2017**
+
```
Comment (gundalow) [inline docs/docsite/rst/roadmap/ROADMAP_2_4.rst:5]: Please add
```

.. toctree::

```

File: docs/docsite/rst/roadmap/ROADMAP_2_4.rst [hunk 2]
```diff
@@ -1,224 +1,220 @@
...
+Administrivia and Process
+-------------------------
+- Starting with 2.4, all items that are deprecated will be removed in 4 major releases unless otherwise stated.
+  - For example: A module that is deprecated in 2.4 will be removed in 2.8
```
Comment (gundalow) [inline docs/docsite/rst/roadmap/ROADMAP_2_4.rst]: Blank line needed here.
In RST blank lines are needed when changing the indent level on bullet points.

File: docs/docsite/rst/roadmap/ROADMAP_2_4.rst [hunk 3]
```diff
@@ -1,224 +1,220 @@
...
+-------------------
+- Ansible Core Engine and Core modules will be tested on Python 3
+- Communicate with Linux distros to provide Ansible running on Python 3
+- Check for Python 3 tests on core modules and create any missing
+
+Ansible-Config
+--------------
+- New yaml format for config
+- Extend the ability of the current config system by adding an ansible-config command and add the following:
+  - Dump existing config settings
```
Comment (gundalow) [inline docs/docsite/rst/roadmap/ROADMAP_2_4.rst]: Blank line needed here.
In RST blank lines are needed when changing the indent level on bullet points.

File: docs/docsite/rst/roadmap/ROADMAP_2_4.rst [hunk 4]
```diff
@@ -1,224 +1,220 @@
...
+Python 3 and beyond
+-------------------
+- Ansible Core Engine and Core modules will be tested on Python 3
+- Communicate with Linux distros to provide Ansible running on Python 3
+- Check for Python 3 tests on core modules and create any missing
+
+Ansible-Config
+--------------
+- New yaml format for config
+- Extend the ability of the current config system by adding an ansible-config command and add the following:
```
Comment (gundalow) [inline docs/docsite/rst/roadmap/ROADMAP_2_4.rst]:
```
``ansible-config``
```

**Review by gundalow — CHANGES_REQUESTED** [review]
Just a few minor changes.
Thanks for doing this

File: docs/docsite/rst/roadmap/ROADMAP_2_4.rst (hunk 1 above)
Comment (dagwieers) [reply docs/docsite/rst/roadmap/ROADMAP_2_4.rst:5]: Done.

**Review by gundalow — APPROVED** [review]

**Review by gundalow — APPROVED** [review]

**Pull request merged**